src/environmentd/src/http/sql.rs

```diff
@@ -1027,36 +1027,49 @@ async fn execute_request<S: ResultSender>(
     let mut stmt_groups = vec![];
 
     match request {
-        SqlRequest::Simple { query } => {
-            let stmts = parse(client, &query)?;
-            let mut stmt_group = Vec::with_capacity(stmts.len());
-            for StatementParseResult { ast: stmt, sql } in stmts {
-                check_prohibited_stmts(sender, &stmt)?;
-                stmt_group.push((stmt, sql.to_string(), vec![]));
+        SqlRequest::Simple { query } => match parse(client, &query) {
+            Ok(stmts) => {
+                let mut stmt_group = Vec::with_capacity(stmts.len());
+                for StatementParseResult { ast: stmt, sql } in stmts {
+                    check_prohibited_stmts(sender, &stmt)?;
```

Comment:
Should this also push the `Err` into `stmt_groups` instead of failing the whole thing?

Reply:
Good thought! I've pushed d8539a28348f2e18e7df437f7a3001f3c50726b3 to do that.